acoustid/api/v2/__init__.py

```
@@ -67,17 +67,17 @@
+    results: set[int] = set()
     for name in args.keys():
         for prefix in prefixes:
             if name == prefix:
-                results.add(None)
+                results.add(-1)
             elif name.startswith(prefix + "."):
                 prefix, suffix = name.split(".", 1)
                 if suffix.isdigit():
                     results.add(int(suffix))
-    return [".%d" % i if i is not None else "" for i in sorted(results)]
+    return [".%d" % i if i != -1 else "" for i in sorted(results)]
```

Ruff (0.8.2)

Use format specifiers instead of percent format

(UP031)